frontend/public/components/utils/headings.tsx

```diff
@@ -171,7 +171,7 @@ export const SectionHeading: React.SFC<SectionHeadingProps> = ({
   style,
   required,
 }) => (
-  <h2 className="co-section-heading" style={style} data-test-section-heading={text}>
+  <h2 className="co-section-heading" style={style} data-test-id={`section-heading`}>
```

Comment:
Do we have to be careful that `data-test-id={`section-heading`}` remains a singleton?

Reply:
I believe in the test I use `.first()`

Reply:
reverted this